From 9af23b930d3115459c8225dbea9aa3a10f3552b8 Mon Sep 17 00:00:00 2001
From: Guilhem IMBERTON <guilhem.imberton@intel.com>
Date: Mon, 15 Oct 2012 16:17:40 +0200
Subject: [PATCH 42/78] CUSTOM_BOARD to TARGET_DEVICE

BZ: 62274

CUSTOM_BOARD is removed. Using original define TARGET_DEVICE

Change-Id: I73466c8faae692eb2656d4791824fc0fc4a2e13d
Signed-off-by: Guilhem IMBERTON <guilhem.imberton@intel.com>
Reviewed-on: http://android.intel.com:8080/70297
Reviewed-by: Seibel, Eric <eric.seibel@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77999
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/AndroidWl12xxCompat.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/AndroidWl12xxCompat.mk b/wl12xx-compat/AndroidWl12xxCompat.mk
index 5ad5c2f..f980d96 100644
--- a/wl12xx-compat/AndroidWl12xxCompat.mk
+++ b/wl12xx-compat/AndroidWl12xxCompat.mk
@@ -2,5 +2,5 @@
 
 .PHONY: build_wl12xx-compat
 build_wl12xx-compat: build_kernel
-	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(CUSTOM_BOARD)
+	TARGET_TOOLS_PREFIX="$(ANDROID_BUILD_TOP)/$(TARGET_TOOLS_PREFIX)" vendor/intel/support/wl12xx-compat-build.sh -c $(TARGET_DEVICE)
 
-- 
1.7.9

